extensions/amp-story-shopping/0.1/amp-story-shopping-attachment.js

```diff
@@ -162,14 +166,17 @@ export class AmpStoryShoppingAttachment extends AMP.BaseElement {
    * @private
    */
   updateTemplate_(shoppingData) {
-    const shoppingDataForPage = this.shoppingTags_.map(
-      (shoppingTag) => shoppingData[shoppingTag.getAttribute('data-product-id')]
+    const pageElement = closestAncestorElementBySelector(
+      this.element,
+      'amp-story-page'
     );
 
+    const shoppingDataPerPage = Object.values(shoppingData[pageElement.id]);
```

Comment:
The annotation for `shoppingData` shows that it is an array. But it is being keyed into by the page ID, suggesting that it's an object. Please update the annotation or the keying logic here (whichever one is incorrect)

Reply:
It was the annotation, I updated it.